woocommerce/woocommerce-ios
# Product description AI: Remove feature flag and local announcement

Closes: #13333 


## Description

Product description AI has been released for more than 1 year so it's time for us to remove the feature flag.
Also, we no longer need the local announcement for the feature, so this PR also removes the related logic. No other feature uses this modal so we can remove the legacy code for now.

## Testing steps

- Log in to a store eligible for Jetpack AI.
- Navigate to Products tab > select an existing product or create a new one.
- On the product form, confirm that Write with AI button is available.


## Screenshots

<img src="https://github.com/user-attachments/assets/0f5765f3-b605-47cf-b967-ae93b99808ef" width=320 />


---
